engine-cdi/src/main/java/org/operaton/bpm/engine/cdi/impl/event/CdiExecutionListener.java

```diff
@@ -21,6 +21,6 @@
  *
  * @author Sebastian Menski
  */
-@Deprecated
+@Deprecated(forRemoval = true, since = "1.0.0-beta-1")
```

Comment:
General: For deprecated classes/methods that have no dedicated replacement I would not mark them for removal. Not everything that is deprecated has to be removed, although likely it will.

To mark new removals this would have to be decided on case-by-case level. Please remove "forRemoval" for such cases. In other words, only add "forRemoval" when the deprecation javadoc states what to use instead.